src/lib/consensus/intf.ml

```diff
@@ -373,7 +373,7 @@ module type S = sig
     module Vrf : sig
       val check :
            context:(module CONTEXT)
-        -> global_slot:Mina_numbers.Global_slot.t
+        -> global_slot:Mina_numbers.Global_slot_since_genesis.t
```

Comment:
This shouid be `Global_slot_since_hard_fork.t`.
ps: #13173 is for this change but it probably won't make it in